contrib/kube-prometheus/build.sh

```diff
@@ -13,3 +13,4 @@ mkdir manifests
 
                                                # optional, but we would like to generate yaml, not json
 jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml; rm -f {}' -- {}
+
```

Comment:
Why do we want this extra newline? The file already correctly ends in a newline (otherwise GitHub would render an red marker)

Reply:
hi @squat . That's a totally fair question. When I do `git clone https://github.com/coreos/prometheus-operator.git` and then open `contrib/kube-prometheus/build.sh`, FWIW there are 2 blank lines at the bottom of the file. In an earlier commit in this PR I removed 1 of the 2 lines, but then the travis-ci build for this PR wasn't passing (I still have the [link](https://travis-ci.org/coreos/prometheus-operator/builds/413568954) to that build - [check/job 2309.1](https://travis-ci.org/coreos/prometheus-operator/jobs/413568955) (which runs the `make format generate-in-docker && git diff --exit-code` command from [prometheus-operator/.travis.yml](https://github.com/coreos/prometheus-operator/blob/master/.travis.yml)) was failing/complaining about a whitespace difference in the portion of `contrib/kube-prometheus/README.md` that has a copy of `build.sh` embedded into it).

I don't understand how/why, but I added the 2nd line back (i.e. this latest commit that you commented on) and now the [latest run of the travis build for this PR](https://travis-ci.org/coreos/prometheus-operator/builds/419195884) is all passing.

@brancz , can this PR please now be merged?